README.md

```diff
@@ -1,4 +1 @@
-paper-radio-group
-=================
-
-See the [component page](http://www.polymer-project.org/docs/elements/paper-elements.html#paper-radio-group) for more information.
+# paper-radio-group
```

Comment:
I've been deleting this from the `readme`. If this is wrong I prooooobably should go back and re-add all of them.